test/alternator/test_returnconsumedcapacity.py

```diff
@@ -126,6 +126,24 @@ def test_long_put(test_table):
     consumed_capacity = response['ConsumedCapacity']
     assert 2 == consumed_capacity["CapacityUnits"]
 
+# WCU is calculated based on 1KB block size.
+# The test validate that when calculating the WCU of a put item
+# over an existing item, the longer of the items is the
```

Comment:
I think you should mention in the comment that by default, Alternator doesn't always implement it like that - it's only guaranteed if read-before-write was done. So this test uses ReturnValues just to force read-before-write, which wouldn't have happened without the ReturnValues flag!